coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20210201-add-column-rollup-created-at.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="Chi" id="20220201-add-column-rollup-created-at">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="rollup_map" columnName="created_at"/>
```

Comment:
I know a lot of tables add created_at, created_meta, created_meta_type and equivalents for updated and deleted. Any reason not to add all here?

Reply:
Those are more like core standard.  I am not pushing core standard to nbe here.